src/Windows/Markdown/Editor/Impl/Preview/Browser/BrowserWindow.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Diagnostics;
+using System.Drawing;
+using System.Reflection;
+using System.Windows.Controls;
+
+namespace Microsoft.Markdown.Editor.Preview.Browser {
+    internal sealed class BrowserWindow {
+        private readonly WebBrowser _webBrowser;
+        private readonly int _zoomFactor;
+        private WebBrowserHostUIHandler _uiHandler;
+
+        public BrowserWindow(WebBrowser webBrowser) {
+            _webBrowser = webBrowser;
+            _zoomFactor = GetZoomFactor();
+            _uiHandler = new WebBrowserHostUIHandler(_webBrowser) { IsWebBrowserContextMenuEnabled = false };
+        }
+
+        public void Init() => Zoom(_zoomFactor);
+
+        private void Zoom(int zoomFactor) {
+            if (zoomFactor == 100) {
+                return;
+            }
+
+            dynamic OLECMDEXECOPT_DODEFAULT = 0;
+            dynamic OLECMDID_OPTICAL_ZOOM = 63;
+            var fiComWebBrowser = typeof(WebBrowser).GetField("_axIWebBrowser2", BindingFlags.Instance | BindingFlags.NonPublic);
+
+            var objComWebBrowser = fiComWebBrowser?.GetValue(_webBrowser);
+            objComWebBrowser?.GetType().InvokeMember("ExecWB", BindingFlags.InvokeMethod, null, objComWebBrowser, new object[] {
```

Comment:
Its huge http://pinvoke.net/default.aspx/Interfaces/IWebBrowser2.html - I guess Mads didn't want to import the whole thing